cmake/AddResources.cmake

```diff
@@ -61,6 +64,14 @@ function(add_resources target)
 				"${android_dir}/assets/${file}"
 				SYMBOLIC
 			)
+		# Assets will already be installed by MACOSX_PACKAGE_LOCATION on macOS
+		elseif(NOT APPLE)
+			# No trailing slash is required or else it will strip the last path
+			# of the directory
+			install(FILES "assets/${file}"
+				DESTINATION "${INSTALL_APPDATADIR}${file_dir}"
+				COMPONENT "${target}"
```

Comment:
Missed a `COMPONENT` to delete here

Reply:
I'm starting to think there's something wrong with my vscode, it keeps opening wrong files and not finding the stuff I grep for. Removed it.